### PR TITLE
167891573 Company Signup Endpoint

### PR DESCRIPTION
**Pull request by femitj**
#### What does this PR do?
This PR implements company signup

#### Description of Task to be completed?
 * write unit test for signup
 * create company 

#### How should this be manually tested?
 - clone the project
 - cd to the project directory
 - checkout to ft-company-signup-endpoint-267891573 branch
 - Run `npm install`
 - Run `npm devstart`
 - test on Postman

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167891573](https://www.pivotaltracker.com/story/show/167891573)

#### Screenshots

Screenshot (62)

Screenshot (63)

#### Questions:
N/A

### CODE REVIEW

**Review by tvpeter — APPROVED** [review]

File: src/controllers/Auth.js [hunk 1]
```diff
@@ -39,13 +44,13 @@ class Auth {
...
         true,
         201,
         'User signup successfully',
-        { user: { email, token }, }
+        { user: { email, token } }
       );
-
       return res.status(response.code).json(response);
     } catch (err) {
       const response = new Response(
```
Comment (thatkazuk1) [inline src/controllers/Auth.js:56]: I think you missed something here. Or I missed something🤷‍♂

**Review by thatkazuk1 — APPROVED** [review]

File: src/controllers/Auth.js (hunk 1 above)
Comment (femitj) [reply src/controllers/Auth.js:56]: You mean i should remove the empty line?, Nothing is missing.

Comment (thatkazuk1) [reply src/controllers/Auth.js:56]: Never mind

**Review by daniellamarr — APPROVED** [review]
Looks good to me. Great job!

**Pull request merged**